python/mxnet/kvstore.py

```diff
@@ -266,8 +283,8 @@ def pull(self, key, out=None, priority=0):
                 self.handle, mx_uint(len(ckeys)), ckeys, cvals, ctypes.c_int(priority)))
 
     def row_sparse_pull(self, key, out=None, priority=0, row_ids=None):
-        """ Pulls a single row_sparse value or a sequence of row_sparse values from the store
-         with specified row_ids.
+        """ Pulls a single RowSparseNDArray value or a sequence of RowSparseNDArray values \
```

Comment:
Why need "\" at the end of the line?

Reply:
causes display problem at API doc. Otherwise, it doesn't treat the second line as part of the first line